pkg/service/collector.go

```diff
@@ -65,19 +65,19 @@ func collectorService(jaeger *v1.Jaeger, selector map[string]string) *corev1.Ser
 			ClusterIP: "",
 			Ports: []corev1.ServicePort{
 				{
-					Name: "zipkin",
+					Name: "http-zipkin",
 					Port: 9411,
 				},
 				{
 					Name: "grpc",
 					Port: 14250,
 				},
 				{
-					Name: "c-tchan-trft",
+					Name: "http-c-tchan-trft",
```

Comment:
Does this port use HTTP? Isn't this tchannel used by the agent?

Reply:
Removing.